app/src/main/java/com/github/geohunt/app/model/database/api/User.kt

```diff
@@ -30,7 +28,11 @@ interface User {
     /**
      * Holds a lazy-reference to a profile picture that can be loaded on a need-to-know basis
      */
-    val profilePicture: LazyRef<Bitmap>
+    val profilePicture: LazyRef<Bitmap>?
+    /**
+     * Hash of the profile picture
+     */
+    val profilePictureHash: Int?
```

Comment:
> this would be straitforward.

No it would not. If you actually check `FirebaseBitmapRef.kt`, you will see that every download is cached. I can not force to redownload a image without doing HUGE modifications that will affect everyone's codebase.